tools/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '2'
+services:
+  vcsim:
+    image: vmware/vcsim
+    ports:
+      - "8989"
+    entrypoint: ["/vcsim", "-host", "${VS_HOSTS:-1}", "-vm", "${VS_VMS:-10}", "-ds", "${VS_DS:-4}", "-l", "0.0.0.0:8989"]
```

Comment:
dumb question , why are the index needed? like the `-1` in `"${VS_HOSTS:-1}"`

Reply:
It's how a default value is set on shell: https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Reply:
Omitting the colon results in a test only for a parameter that is unset

Reply:
oh nice trick , i though it was string handling with index 👍